tools/README.md

```diff
@@ -0,0 +1,35 @@
+# How to Build
+
+## Requirements
+
+Environment Variables:
+- `AWS_ACCESS_KEY`
+- `AWS_SECRET_KEY`
+- `GITHUB_API_TOKEN`
+
+## LogDNA Agent for Debian Systems
+LogDNA Agent can be built and released for Debian systems using `debian.sh` script which has the following steps:
+1. Install Dependencies:
+	* Install `nexe` which is for compiling the files into the executable;
+	* Install `fpm` for packaging it up and `deb-s3` for pushing the packages into the specified S3 bucket;
```

Comment:
Semicolons at the end of these sentences is weird.  Just remove them, or add a period to match the last bullet point.

Reply:
@darinspivey, done!